cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -31,6 +31,9 @@ Replacement syntax for deprecated and removed features are also indicated.
 [options="header"]
 |===
 | Feature     | Type | Change | Details
+| `REVOKE MATCH` | Syntax | Added | The `MATCH` privilege can now be revoked
+| `REVOKE ...` | Functionality | Restricted | No longer revokes sub-privileges when revoking a compound privilege
+| `ALL DATABASE PRIVILEGE` | Functionality | Restricted | No longer includes the privileges `START DATABASE` and `STOP DATABASE`
```

Comment:
- Should we also add something here about the compound not being split when granted?
- And that we can revoke a sub-privilege even if the compound exists?

Reply:
Perhaps we add a row for `SHOW PRIVILEGES` that says it is changed to show privileges more like those written (ie. if writing a compound we list that, if writing a leaf we list that). I think the internal splitting, not splitting does not need to be described, but the grants and the show are both surfaces, and show now changes because of this.